hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -4,6 +4,8 @@
 {
   flow: formal
 
+  sub_flow: fpv
```

Comment:
Is this needed? Each individual cfg below imports `common_fpv_cfg.hjson` which sets the `sub_flow`.

Reply:
That is a good point Sri! Thanks for pointing that out.
I believe the answer is yes -> I think this is a similar reason why we need to set `flow: formal` above. I found the master(or called the primary) cfg does not have this parameter. So this will throw an error in line 31 of FormalCfg.py
```
self.sub_flow = hjson_data['sub_flow']
```

Reply:
Hmm, we should clean that up a bit. This is ok for now.

Reply:
Yeah I will add a TODO here!